capstone-team-1/server/emaillNotification.js

```diff
@@ -0,0 +1 @@
+g;
```

Comment:
wut

Reply:
We had some merge issues and code commit issues that we were attempting to push files that had no changes in them and we added a single line to simply trigger the need to update the file.